pkg/postage/listener/listener.go

```diff
@@ -327,19 +321,25 @@ func (l *listener) Listen(from uint64, updater postage.EventUpdater, initState *
 
 	go func() {
 		err := listenf()
+		defer func() {
+			select {
+			case _, ok := <-synced:
+				if ok {
+					synced <- err
+				}
+			default:
+				synced <- err
+			}
+			closeOnce.Do(func() { close(synced) })
+		}()
 		if err != nil {
 			if errors.Is(err, context.Canceled) {
 				// context cancelled is returned on shutdown,
 				// therefore we do nothing here
 				return
 			}
 			l.logger.Errorf("failed syncing event listener, shutting down node err: %v", err)
-			if l.shutdowner != nil {
-				err = l.shutdowner.Shutdown()
-				if err != nil {
-					l.logger.Errorf("failed shutting down node: %v", err)
-				}
-			}
+			close(l.running) // trigger shutdown in start.go
```

Comment:
there are two `close` calls for this channel (one here and one on the actual shutdown method). this will panic at one point or another

Reply:
removed the leftover `close`